src/components/textarea/TextArea.tsx

```diff
@@ -3,8 +3,10 @@ interface TextAreaInterface {
   name: string
   placeholder: string
   value?: string
+  error?: string | null
```

Comment:
not needed 🤔

Reply:
![image](https://github.com/nerubia/kh360-react/assets/91468846/b222e7a2-d93a-4b93-970c-b956a7a3589e)
Nageerror po sha dito sa EvaluationsCriteria.tsx pag niremove ko po sha

Reply:
ah I see